pkg/gardenlet/controller/seed/seed/reconciler.go

```diff
@@ -60,8 +61,13 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
+	gardenCtx, cancel := controllerutils.GetMainReconciliationContext(ctx, r.Config.Controllers.Seed.SyncPeriod.Duration)
```

Comment:
I don't think that 1 minute context will be enough for the seed reconciler. Imho we should leave the seed and shoot reconcilers' contexts without timeouts for now.